cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -67,16 +69,24 @@
   private final SessionManager _sessionManager;
   private final long _maxBlockMs;
   private final ThreadLocal<Integer> _asyncOperationStep;
+  private final HashMap<EndPoint, Meter> _requestMeter = new HashMap<>();
+  private final HashMap<EndPoint, Timer> _requestExecutionTimer = new HashMap<>();
```

Comment:
1. `private final HashMap<EndPoint, Meter>` -> `private final Map<EndPoint, Meter> `
2. `private final HashMap<EndPoint, Timer>` -> `private final Map<EndPoint, Timer>`
3. let's initialize the private variables in the constructor.